plugins/package-managers/go/src/funTest/assets/projects/synthetic/gomod/go-as-dep-with-transitive-deps-expected-output.yml

```diff
@@ -109,16 +43,16 @@ packages:
       algorithm: ""
   vcs:
     type: "Git"
-    url: "https://github.com/golang/snappy"
-    revision: "2a8bb927dd31d8daada140a5d09578521ce5c36a"
+    url: "https://github.com/klauspost/shutdown2"
+    revision: "8cffc584385a89235a56e64a8f610ced202c1da0"
     path: ""
   vcs_processed:
     type: "Git"
-    url: "https://github.com/golang/snappy.git"
-    revision: "2a8bb927dd31d8daada140a5d09578521ce5c36a"
+    url: "https://github.com/klauspost/shutdown2.git"
+    revision: "8cffc584385a89235a56e64a8f610ced202c1da0"
     path: ""
-- id: "Go::github.com/klauspost/compress:1.13.6"
-  purl: "pkg:golang/github.com/klauspost/compress@1.13.6"
+- id: "Go::golang.org/x/net:0.37.0"
```

Comment:
Just double-checking: Is it really correct that this dependencies appears nowhere under `scopes` above?

Reply:
Good catch, no idea where this comes from. Need to investigate. `go mod tidy` has added it to `go.mod`.

Reply:
Shall we extract this into a `bug` ticket and proceed with this PR?

Reply:
Yes, let's do that: https://github.com/oss-review-toolkit/ort/issues/10099